### PR TITLE
Wordpressinapp

### PR DESCRIPTION
**Pull request by gsacavdm**
- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog
* Add template for Wordpress via App Service with MySQL in App

### CODE REVIEW

File: wordpress-app-service-mysql-inapp/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "siteName": {
+            "type": "string",
+            "metadata": {
+                "description": "App service site name"
+            }
+        },
+        "hostingPlanName": {
```
Comment (bmoore-msft) [inline wordpress-app-service-mysql-inapp/azuredeploy.json]: add a defaultValue or just move this to a variable

File: wordpress-app-service-mysql-inapp/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Web/serverfarms",
+            "sku": {
+                "name": "[parameters('sku')]",
+                "capacity": 1
+            },
+            "name": "[parameters('hostingPlanName')]",
+            "apiVersion": "2015-08-01",
```
Comment (bmoore-msft) [inline wordpress-app-service-mysql-inapp/azuredeploy.json]: update apiVersion

File: wordpress-app-service-mysql-inapp/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+            "location": "[resourceGroup().location]",
+            "properties": {}
+        },
+        {
+            "type": "Microsoft.Web/sites",
+            "name": "[parameters('siteName')]",
+            "apiVersion": "2015-08-01",
```
Comment (bmoore-msft) [inline wordpress-app-service-mysql-inapp/azuredeploy.json]: update apiVersion

File: wordpress-app-service-mysql-inapp/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+                            "value": "--max_allowed_packet=16M"
+                        }
+                    ]
+                }
+            },
+            "resources": [
+                {
+                    "type": "sourcecontrols",
+                    "name": "web",
+                    "apiVersion": "2015-08-01",
```
Comment (bmoore-msft) [inline wordpress-app-service-mysql-inapp/azuredeploy.json]: update apiVersion

**Comment by gsacavdm** [comment]
Fixed @bmoore-msft

**Pull request merged**